source/screens/caseScreens/EKBCases.js

```diff
@@ -191,7 +192,7 @@ const EKBCases = ({ navigation, route }) => {
     <CaseArchiveWrapper>
       <Header title="Ekonomiskt bistånd" themeColor="purple" />
       <Container>
-        <StatusComponent />
+        {latestCase && <StatusComponent />}
```

Comment:
What is shown if there's no latest case?

Reply:
Good point (y)